types/test-console/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for test-console 1.1
+// Project: https://github.com/jamesshore/test-console
+// Definitions by:  Roberto Soares <https://github.com/roberto>
+//                  Pedro Guidoux <https://github.com/guidoux>
+//                  Guilherme Moretti <https://github.com/gbmoretti>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export const stdout: TestStream;
+export const stderr: TestStream;
+
+type Output = ReadonlyArray<string>;
+type OutputCallback = (output: Output) => void;
+type NoOutputCallback = () => void;
+type Restore = () => void;
+
+interface Options {
+    isTTY?: boolean;
+}
+
+export interface Inspector {
+    output: Output;
+    restore: Restore;
+}
+
+export interface TestStream {
+    inspect(options?: Options): Inspector;
+    inspectSync(fn: OutputCallback): Output;
+    inspectSync(options: Options, fn?: OutputCallback): Output;
+    ignore(options?: Options): Restore;
+    ignoreSync(fn: NoOutputCallback): void;
+    ignoreSync(options: Options, fn?: NoOutputCallback): void;
+}
+
+export {};
```

Comment:
You don't need this statement

Reply:
@sheetalkamat thanks for reviewing this!

About this statement, if I remove I got:
```
strict-export-declare-modifiers  All declarations in this module are exported automatically.
Prefer to explicitly write 'export' for clarity.
If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting.
See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```
In the lines 11, 12, 13, 14 and 16 (not exported elements). Should I export all of them instead of adding `export {}`?

Reply:
You can add export on all types instead.